### PR TITLE
Update clients

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add SBA, timing, viecuri
- [x] Make a helper and reduce locale specific files, because 🇬🇧 is always same order as 🇳🇱

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -605,6 +605,18 @@ def capitalize(tagname)
...
 
+  # Get clients
+  def clients
+    all_clients =
+      case root_locale
+      when :de
+        data["clients_de"]
+      else
+        data["clients"]
+      end
```
Comment (fatboypunk) [inline config.rb]: This is just an `if ... else` right?

Comment (fatboypunk) [reply config.rb]: This would also work
```rb
all_clients = :de ? data["clients_de"] : data["clients"]
```

Comment (RonaldDijkstra) [reply config.rb]: Done, thx!

File: config.rb [hunk 2]
```diff
@@ -605,6 +605,12 @@ def capitalize(tagname)
     tagname.gsub(/-[a-z]/, &:upcase)
   end
 
+  # Get clients
+  def clients
+    all_clients = root_locale == :de ? data["clients_de"] : data["clients"]
+    all_clients.clone
```
Comment (sn3p) [inline config.rb]: Curious, why does this needs to be cloned? We're not manipulating the data arrays right?

Comment (RonaldDijkstra) [reply config.rb]: Well was in the original code in the template so I thought that it's needed. Will have a look if that really is the case :wink:

**Pull request merged**